### PR TITLE
Add new APIs - placeholder

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/osv2-apis.adoc [hunk 1]
```diff
@@ -474,6 +479,53 @@ This request returns a response similar to the following:
...
+==== Example: Retrieve Usage Metrics by Region
+
+To view Object Store usage statistics for a region,
+submit a `curl` command similar to the following, 
+replacing ORG_ID with your organization ID, ENV_ID with your environment ID,
+and BEARER_TOKEN with your bearer token:
+
+[source,console,linenums]
+----
+curl -X GET \
```
Comment (bahui80) [inline modules/ROOT/pages/osv2-apis.adoc]:
```
curl -X GET \
  'https://object-store-stats.anypoint.mulesoft.com/api/v1/organizations/ORG_ID/environments/ENV_ID/regions/REGION_ID?startDate=2020-06-13T17%3A51%3A54.000Z&endDate=2020-09-13T17%3A51%3A54.000Z&period=1month' \
  -H 'authorization: Bearer BEARER_TOKEN'
  ```

File: modules/ROOT/pages/osv2-apis.adoc [hunk 2]
```diff
@@ -474,6 +479,53 @@ This request returns a response similar to the following:
 ]
 ----
 
+==== Example: Retrieve Usage Metrics by Region
+
+To view Object Store usage statistics for a region,
+submit a `curl` command similar to the following, 
+replacing ORG_ID with your organization ID, ENV_ID with your environment ID,
```
Comment (bahui80) [inline modules/ROOT/pages/osv2-apis.adoc]: add here replacing REGION_ID with the region in which to get metrics from

File: modules/ROOT/pages/osv2-apis.adoc [hunk 3]
```diff
@@ -474,6 +479,53 @@ This request returns a response similar to the following:
...
+----
+curl -X GET \
+ 
+----
+
+This request returns a response similar to the following:
+
+[source,json,linenums]
+----
+[
```
Comment (bahui80) [inline modules/ROOT/pages/osv2-apis.adoc:579]: Output for all metrics API have the same structure: 
```
[
 {"timeStamp":"2020-06-01T00:00:00Z","objectStoreRequestCount":3},
 {"timeStamp":"2020-07-01T00:00:00Z","objectStoreRequestCount":30},
 {"timeStamp":"2020-08-01T00:00:00Z","objectStoreRequestCount":1},
 {"timeStamp":"2020-09-01T00:00:00Z","objectStoreRequestCount":25}
]
```

File: modules/ROOT/pages/osv2-apis.adoc [hunk 4]
```diff
@@ -474,6 +479,53 @@ This request returns a response similar to the following:
...
+==== Example: Retrieve Usage Metrics by Store
+
+To view Object Store usage statistics for a store,
+submit a `curl` command similar to the following, 
+replacing ORG_ID with your organization ID, ENV_ID with your environment ID,
+and BEARER_TOKEN with your bearer token:
+
+[source,console,linenums]
+----
+curl -X GET \
```
Comment (bahui80) [inline modules/ROOT/pages/osv2-apis.adoc]:
```
curl -X GET \ 'https://object-store-stats.anypoint.mulesoft.com/api/v1/organizations/ORG_ID/environments/ENV_ID/regions/REGION_ID/stores/STORE_ID?startDate=2020-06-13T17%3A51%3A54.000Z&endDate=2020-09-13T17%3A51%3A54.000Z&period=1month' \ -H 'authorization: Bearer BEARER_TOKEN'
```

File: modules/ROOT/pages/osv2-apis.adoc [hunk 5]
```diff
@@ -474,6 +479,53 @@ This request returns a response similar to the following:
...
+ 
+----
+
+This request returns a response similar to the following:
+
+[source,json,linenums]
+----
+[
```
Comment (bahui80) [inline modules/ROOT/pages/osv2-apis.adoc:611]: Output for all metrics API have the same structure: 
```
[
 {"timeStamp":"2020-06-01T00:00:00Z","objectStoreRequestCount":3},
 {"timeStamp":"2020-07-01T00:00:00Z","objectStoreRequestCount":30},
 {"timeStamp":"2020-08-01T00:00:00Z","objectStoreRequestCount":1},
 {"timeStamp":"2020-09-01T00:00:00Z","objectStoreRequestCount":25}
]
```

File: modules/ROOT/pages/osv2-apis.adoc [hunk 6]
```diff
@@ -474,6 +479,53 @@ This request returns a response similar to the following:
...
+
+
+
+For the list of CloudHub deployment regions, see xref:runtime-manager::cloudhub-networking-guide.adoc#regional-services[Regional Services].
+
+==== Example: Retrieve Usage Metrics by Store
+
+To view Object Store usage statistics for a store,
+submit a `curl` command similar to the following, 
+replacing ORG_ID with your organization ID, ENV_ID with your environment ID,
```
Comment (bahui80) [inline modules/ROOT/pages/osv2-apis.adoc]: Add here same as before for region  id and also adding store id which to get metrics from

**Review by bahui80 — APPROVED** [review]
LGTM!

**Review by sathya0 — APPROVED** [review]

**Pull request merged**